app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -64,7 +64,7 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         savedInstanceState: Bundle?
     ): View {
         val view = super.onCreateView(inflater, container, savedInstanceState)
-        view.browserLayout.transitionName = "$TAB_ITEM_TRANSITION_NAME${getSessionById()?.id}"
+
         startPostponedEnterTransition()
```

Comment:
Do we still need the postponeEnterTransition and startPostponedEnterTransition if we're removing the shared element transition from browser?

Reply:
@ekager Potentially. I don't need it for the animations I'm doing right now, but I'm going to have a follow up PR that moves the browser toolbar up with Browser -> Search transition. So I'd like to keep it for now.